config/pmd.xml

```diff
@@ -89,20 +89,16 @@
      <property name="annotationPattern" value="[A-Z][a-zA-Z0-9]*" />
      <property name="utilityClassPattern" value="[A-Z][a-zA-Z0-9]*Util" />
      <!-- We can not brake compatibility with previous versions. -->
-     <property name="violationSuppressXPath"
-               value="//ClassOrInterfaceDeclaration
-                          [@Image='AbstractClassNameCheck' or @Image='AutomaticBean']"/>
-    </properties>
-  </rule>
-  <rule ref="category/java/codestyle.xml/ClassNamingConventions">
-    <properties>
      <!-- Definitions and XmlLoader.LoadExternalDtdFeatureProvider aren't utility classes.
           JavadocTokenTypes and TokenTypes aren't utility classes.
             They are token definition classes. Also, they are part of the API. -->
      <property name="violationSuppressXPath"
-               value="//ClassOrInterfaceDeclaration[@Image='Definitions'
-                   or @Image='JavadocTokenTypes' or @Image='TokenTypes'
-                   or @Image='LoadExternalDtdFeatureProvider']"/>
+               value="//ClassOrInterfaceDeclaration
+                          [@Image='AbstractClassNameCheck' or @Image='AutomaticBean']
```

Comment:
New images but no new descriptions to follow them.

Reply:
diff is not very friendly for review, please review result file - there two comments over rule.

Reply:
https://github.com/checkstyle/checkstyle/blob/9e0292dad59b866c859c3e50738cbbf65fdbc6b1/config/pmd.xml#L91
I assume the first comment is for AbstractClassNameCheck and AutomaticBean? If so, shouldn't we specify the names in the description so the connection is clear?

Reply:
I rewrote comment a bit.
I removed AbstractClassNameCheck from xpath.

please take a look now.